gpii/node_modules/flowManager/src/FlowManager.js

```diff
@@ -323,3 +323,37 @@ gpii.flowManager.local.savePreferences = function (that, gpiiKey, preferences) {
         that.events.preferencesSavedSuccess.fire();
     }
 };
+
+/**
+ * When the flag "suppressHttpEndpoints" is turned on, remove unneeded http endpoints from the local flow manager.
+ * When this flag is turned off, add these endpoints.
+ * @param {Boolean} suppressHttpEndpoints - The flag to indicate whether to remove unneeded http endpoints from LFM.
+ * @return {String} If the flag is true, return undefined. Otherwise, return the gradename that attach extra endpoints.
+ */
+gpii.flowManager.local.httpEndpointsGradeFinder = function (suppressHttpEndpoints) {
```

Comment:
Better to express this via the contextAwareness feature https://docs.fluidproject.org/infusion/development/ContextAwareness.html#example-contextawareness-record than writing manual logic